tools/misc/calc_timeprop_spectrum

```diff
@@ -15,17 +15,24 @@ import argparse
 import numpy as np
 from scipy import constants
 
-USAGE = """Reads output from TD calculation (after kick) and produces
-absorption spectra. Needs mux.dat, muy.dat and muz.dat files in working directory."""
+USAGE = """%(prog)s -d xx
+
+Reads output from TD calculation (after kick) and produces absorption spectra.
+
+Needs mux.dat, muy.dat and muz.dat files in working directory."""
 
 def main():
-    parser = argparse.ArgumentParser(description=USAGE)
-    parser.add_argument("tau", metavar="DAMPING",
+    parser = argparse.ArgumentParser(usage=USAGE)
+    parser.add_argument("-d", "--damping", dest="tau",
                         help="damping constant in fs")
-    parser.add_argument("field", metavar="FIELD",
+    parser.add_argument("-f", "--field", dest="field",
                         help="field intensity in volts/angstrom")
 
-    args = parser.parse_args()
+    options, args = parser.parse_known_args()
```

Comment:
Why not using `parser_args()` instead? It would automatically take care to check, that no positional arguments are provided.

Reply:
I changed it to `parse_args()`. My idea was simply to keep the message.